keanu-python/keanu/vertex.py

```diff
@@ -105,13 +104,25 @@ def __ceil__(self):
         return kn.generated.vertex.Ceil(self)
 
 class Vertex(JavaObjectWrapper, VertexOps):
-    def __init__(self, ctor, *args):
-        super(Vertex, self).__init__(ctor, *(Vertex.__parse_args(*args)))
+    def __init__(self, ctor=None, *args, java_vertex=None):
+        if java_vertex is None:
+            super(Vertex, self).__init__(ctor(*(Vertex.__parse_args(*args))))
+        else:
+            super(Vertex, self).__init__(java_vertex)
+
+    def __hash__(self):
+        return hash(self.get_id())
 
     def observe(self, v):
         from keanu.tensor import Tensor
         self.unwrap().observe(Tensor(v).unwrap())
 
+    def get_connected_graph(self):
+        return Vertex.to_python_set(self.unwrap().getConnectedGraph())
```

Comment:
Clearer to have `self` and not `Vertex` if it's in the same class

Reply:
self is associated with the instance in python, so I prefer using `Vertex` for calling static methods